docs/docs/02-configuration/10-derived-type-mapping.md

```diff
@@ -0,0 +1,32 @@
+# Derived types and interfaces
+
+Mapperly supports interfaces and base types as mapping sources and targets,
+but Mapperly needs to know which derived types exist.
+This can be configured with the `MapDerivedTypeAttribute`:
+
+```csharp
+[Mapper]
+public static partial class ModelMapper
+{
+    // highlight-start
+    [MapDerivedType<Audi, AudiDto>] // for c# language level ≥ 11
+    [MapDerivedType(typeof(Porsche), typeof(PorscheDto))] // for c# language level < 11
+    // highlight-end
+    public static partial CarDto MapCar(Car source);
+}
+
+abstract class Car {}
+class Audi : Car {}
+class Porsche : Car {}
+
+abstract class CarDto {}
+class AudiDto : CarDto {}
+class PorscheDto : CarDto {}
+```
+
+Mapperly implements a type switch with an arm for each source type.
```

Comment:
Is "with an arm" correct? Even if it is, I don't think that is a statement that a lot of developers will understand.

Maybe showing the generated code is easier in this case? Does not have to be the complete, accurate generated code, just a sample

Reply:
Fixed.